styleguide/source/_patterns/01-molecules/06-components/20-testimonial/_testimonial.scss

```diff
@@ -0,0 +1,78 @@
+.testimonial {
+  position: relative;
+  width: 100%;
+  max-width: 1200px;
+  margin-bottom: 24px;
+
+  @include breakpoint($bp-med) {
+    display: flex;
+    flex-direction: row-reverse;
+    align-items: stretch;
+
+    &.testimonial-left {
+      flex-direction: row;
+    }
+  }
+}
+
+.testimonial_image {
+  display: flex;
+  flex-direction: column;
+
+  @include breakpoint($bp-med) {
+    min-width: 50%;
+    flex-basis: 50%;
+  }
+}
+
+.testimonial_image span {
+  margin: auto 0;
+  min-height: calc((100vw - 60px) / 2 * 3);
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+
+  @include breakpoint($bp-small) {
+    min-height: calc((100vw -  60px) / 3 * 2);
+  }
+
+  @include breakpoint($bp-med) {
+    min-height: calc(((100vw -  60px) / 2) / 3 * 2);
+  }
+
+  @include breakpoint($bp-large) {
+    min-height: calc(((1260px -  60px) / 2) / 3 * 2);
+  }
+}
+
+.testimonial_text {
+  padding: 30px;
+
+  @include breakpoint($bp-med max-width) {
```

Comment:
I'm seeing some unusual behavior at exactly 900px in size. (see screenshot). 

![screen shot 2017-06-26 at 11 02 00 am](https://user-images.githubusercontent.com/28711067/27549368-53ed734c-5a61-11e7-90c6-fd6134e0dce6.png)

I think we can clear this up by just organizing the css a little more according to a mobile-first pattern.  Like:

```
.testimonial_text {
  bottom: 0;
  background-color: rgba(255, 255, 255, 0.45);
  padding: 15px;
  position: absolute;

  @include breakpoint($bp-small) {
    padding: 30px;
  }
  
  @include breakpoint($bp-med) {
    position: relative;
  }
}
```

I added the extra breakpoint just because I think the tablet view can support the extra padding. But this should make all our requirements stack properly between the different viewport sizes.